instances/deviceType/vibratingMicrotome.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
```

Comment:
modified a bit the definition and changed the name to "vibrating microtome", because apparently "vibratome" and "compresstome" are not generic names but registered trademarks of different manufacturers

Reply:
I agree to add "vibratome" as a synonym

"typically cut at" does not cover it since you can cut much larger sections with a vibratome.

-Let's leave the slice thickness limitations then out for "vibrating microtome" and "microtome"